test cases/common/178 bothlibraries/gen.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import argparse
+import shutil
+
+
+def main() -> None:
```

Comment:
Ooor we could just not bother type-annotating the least important part of an extremely trivial copy.py script. Which we should just be copying from one of the other 19 cp.py / copyfile.py implementations in `test cases/`, which generally do not go to the overhead of importing argparse for a non-user-facing script...